new-games-to-db/start.py

```
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
```

Ruff (0.11.9)

Shebang is present but file is not executable

(EXE001)